src/backend/web/routes/admin.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const { UI } = require('bull-board');
+
+const router = express.Router();
+
+router.use('/queues', UI);
+
+module.exports = router;
```

Comment:
not needed

Reply:
I do believe this is all currently needed to make the routes work. I followed this guide (https://www.youtube.com/watch?v=paNikhYqdz0&t=1s) and checked the express documentation about express routing (https://expressjs.com/en/guide/routing.html) and their example at the very bottom of the screen also does it in this way.

I also resolved some of the other comments made in this file since they were all the same.

Reply:
@varshannagarajan  yup you're right! I stand corrected :)